app/src/main/java/com/github/dedis/student20_pop/utility/network/HighLevelClientProxy.java

```diff
@@ -141,9 +139,9 @@ public LowLevelClientProxy lowLevel() {
     /**
      * Send an open roll call message
      *
-     * @param laoId id of the lao
+     * @param laoId      id of the lao
      * @param rollCallId id of the roll call
```

Comment:
Why isn't rollCallId offset like the rest ?

Reply:
What do you mean by that ?